pkg/kube/clusterregistry/secretcontroller.go

```diff
@@ -0,0 +1,263 @@
+// Copyright 2018 Istio Authors
```

Comment:
might find a better home for this file rather than pkg/kube/clusterregistry as we are no longer using the clusterregistry code.